app/views/layouts/_head.html.erb

```diff
@@ -22,6 +22,11 @@
 <%= favicon_link_tag "#{AppSettings['design.favicon']}" %>
 <%= cloudinary_js_config %>
 
+<% if rtl_locale?(params["locale"]) %>
+<!-- Load Bootstrap RTL theme from RawGit -->
+<link rel="stylesheet" href="//cdn.rawgit.com/morteza/bootstrap-rtl/v3.3.4/dist/css/bootstrap-rtl.min.css">
+<% end %>
+
```

Comment:
For now, we just load the bootstrap RTL css if the current locale is recognized to be RTL.  Add any additional CSS here.